js/battledata.js

```diff
@@ -847,7 +847,12 @@ var Tools = {
 			facing = 'back';
 		}
 
-		var animationData = window.BattlePokemonSprites && BattlePokemonSprites[pokemon.speciesid];
+		var animationData = {};
+		if (options.gen === 5 && typeof(BattlePokemonSpritesBW) !== 'undefined') {
```

Comment:
Convention is to use `window.BattlePokemonSpritesBW` instead of `typeof`. Either way, you're using `typeof` against convention - note that `typeof` is an operator, not a function.